test/lib/mayaUsd/render/mayaToHydra/testFlowViewportAPI.py

```diff
@@ -102,7 +102,7 @@ def test_AddingPrimitives(self):
             self.setHdStormRenderer()
             self.assertSnapshotClose("add_VP2AndThenBackToStorm.png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
 
-            #Finish by a File New command
+            #Finish by a File New command to check that it's not crashing when cleaning up everything
```

Comment:
Sorry, should have caught this earlier, since you're calling file new after each test, you should factor this out into a tearDown() method:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown